articles/aws-api-setup.md

```diff
@@ -1,70 +1,104 @@
 
   ![](/media/articles/aws-api-setup/aws-api-setup-2.png)
 
-<% if (account.userName) { %>
-3. Download the metadata from <a href="https://${account.namespace}/samlp/metadata/${account.clientId}">https://${account.namespace}/samlp/metadata/${account.clientId}</a> and upload it here.
-<% } else { %>
-3. Download the metadata from `https://${account.namespace}/samlp/metadata/${account.clientId}` and upload it here.
-<% } %>
+3. Download your metadata document from <% if (account.userName) { %><https://${account.namespace}/samlp/metadata/${account.clientId}><% } else { %>`https://${account.namespace}/samlp/metadata/${account.clientId}`<% } %>.
+
+  Click **Choose File** and browse to the metadata document you just downloaded. Click **Next Step**:
 
   ![](/media/articles/aws-api-setup/aws-api-setup-3.png)
 
-4. The SAML Provider was created. You can close the modal.
+4. Verify your provider information and click **Create**:
 
   ![](/media/articles/aws-api-setup/aws-api-setup-4.png)
 
-## Creating Roles
+## Create a Role
 
-Now, you have to create a role that will have one or more policies associated. You can create as many roles as you want.
+Now create a role with one or more associated policies. (You can create as many roles as required.)
 
-1. Go to **Roles**.
+1. On the AWS IAM Console, click on **Roles** in the left menu and then click **Create New Role**:
 
   ![](/media/articles/aws-api-setup/aws-api-setup-5.png)
 
-2. Click on **Create New Role** and give it an arbitrary name.
+2. Provide a name for your new role and click **Next Step**:
 
   ![](/media/articles/aws-api-setup/aws-api-setup-6.png)
 
-3. Select or enter the following values.
+3. Select **Role for Identity Provider Access**. Then select **Grant API access to SAML providers** and click **Next Step**:
 
-  * **SAML Provider**: the provider you've just created
-  * **Attribute**: `SAML:iss`
-  * **Value**: `urn:${account.namespace}`
+  ![](/media/articles/aws-api-setup/aws-api-setup-7.png)
 
+4. Select or enter the following values:
 
-  ![](/media/articles/aws-api-setup/aws-api-setup-7.png)
+  * **SAML Provider**: the provider you just created
+  * **Attribute**: `SAML:iss`
+  * **Value**: `urn:${account.namespace}`
 
-4. This is just a confirmation. Click on **Continue**.
+  and click **Next Step**:
 
   ![](/media/articles/aws-api-setup/aws-api-setup-8.png)
 
-5. This is where you assign permissions to this role. In this example, we are creating a policy that give full access to the S3 resource `YOUR_BUCKET/<%= '${saml:sub}' %>`. This will be evaluated on runtime and replaced with the `user_id` of the logged in user.
+5. On the Verify Role Trust page, accept the Policy Document as provided. Click **Next Step**:
 
   ![](/media/articles/aws-api-setup/aws-api-setup-9.png)
 
-  > For more information about what AWS APIs are supported: <http://docs.aws.amazon.com/IAM/latest/UserGuide/Using_SpecificProducts.html>
+6.  Skip the **Attach Policy** step for now. Click **Next Step**.
```

Comment:
This is the point that I was talking about for the attach policy. Wasn't sure if this was a specific example, I didn't think it was, so I'd instead say....

At **Attach Policy** either select a pre-defined policy to attach or define a custom policy as defined in the next section. Click **Next Step**